src/renderer/components/+extensions/extensions.tsx

```diff
@@ -514,7 +515,13 @@ export class Extensions extends React.Component<Props> {
           <section>
             <h1>Extensions</h1>
 
-            <Notice/>
+            <Notice>
```

Comment:
We probably should expose `Notice` to extensions? (story for another PR)

Reply:
Yes, for sure.